server/src/main/java/io/crate/planner/operators/InsertFromValues.java

```diff
@@ -486,11 +498,13 @@ public List<CompletableFuture<Long>> executeBulk(DependencyCarrier dependencies,
         );
     }
 
-    private void checkConstraintsOnGeneratedSource(@Nullable IndexItem indexItem,
-                                                   String indexName,
-                                                   DocTableInfo tableInfo,
-                                                   PlannerContext plannerContext,
-                                                   HashMap<String, Consumer<IndexItem>> validatorsCache) throws Throwable {
+    public static void checkConstraintsOnGeneratedSource(@Nullable IndexItem indexItem,
```

Comment:
Not dedicated to this change, but the existing name is a bit missleading nowadays.
```suggestion
    public static void checkConstraints(@Nullable IndexItem indexItem,
```